qiskit_ibm_runtime/sampler.py

```diff
@@ -146,6 +153,40 @@ def __init__(
                     log levels are: ``DEBUG``, ``INFO``, ``WARNING``, ``ERROR``, and ``CRITICAL``.
                     The default level is ``WARNING``.
 
+            skip_transpilation: (DEPRECATED, use transpilation_settings.skip_transpilation instead)
+                Transpilation is skipped if set to True. False by default.
+
+            transpilation_settings: Qiskit transpiler settings. The transpilation process converts
+                operations in the circuit to those supported by the backend, swaps qubits with the
+                circuit to overcome limited qubit connectivity and some optimizations to reduce the
+                circuit's gate count where it can.
+
+                * skip_transpilation: Transpilation is skipped if set to True.
+                    False by default.
+
+                * optimization_settings:
+
+                    * level: How much optimization to perform on the circuits.
+                        Higher levels generate more optimized circuits,
+                        at the expense of longer transpilation times.
+                        * 0: no optimization
+                        * 1: light optimization
+                        * 2: heavy optimization
+                        * 3: even heavier optimization
+                        If ``None``, level 1 will be chosen as default.
+
+            resilience_settings: Using these settings allows you to build resilient algorithms by
+                leveraging the state of the art error suppression, mitigation and correction techniques.
+
+                * level: How much resilience to build against errors.
+                    Higher levels generate more accurate results,
+                    at the expense of longer processing times.
+                    * 0: no resilience
+                    * 1: light resilience
+                    * 2: heavy resilience
+                    * 3: even heavier resilience
+                    If ``None``, level 1 will be chosen as default.
```

Comment:
So for the sampler this should probably not be set by default.  The reason is because then the returned results can contain negative values.  I can not use these values in many places where a true probably distribution can.  One would have to consume them for expectation values, or map onto the closet probability distribution.    Most our users are not accustom to having to do this.  Additionally there is no good way to do the expectation values from the quasi-distributions in Qiskit at present.  So taken together, the default should probably be 0 here.

Reply:
Fixed in https://github.com/Qiskit/qiskit-ibm-runtime/pull/368/commits/c2227151eab657431f7a64c03f5cdb5930ff3328